packages/grid/src/vaadin-grid.js

```diff
@@ -644,7 +644,7 @@ class Grid extends ElementMixin(
   }
 
   /** @private */
-  _createCell(tagName) {
+  _createCell(tagName, isCellButton) {
```

Comment:
We could consider passing the column as context, then we don't have to invent new names here, and it could be useful in the future, or for sub-classes.

Reply:
Done.